example/ViewPagerAndroidExample.android.js

```diff
@@ -179,6 +179,9 @@ export default class ViewPagerAndroidExample extends React.Component {
           onPageSelected={this.onPageSelected}
```

Comment:
I think, we don't want to split it into two files. `ViewPagerAndroidExample` should be for both platforms.